src/include/daos_srv/vos.h

```diff
@@ -274,12 +274,13 @@ vos_pool_destroy(const char *path, uuid_t uuid);
  * \param path	[IN]	Path of the memory pool
  * \param uuid	[IN]    Pool UUID
  * \param flags [IN]	Pool open flags (see vos_pool_open_flags)
+ * \param metrics[IN]	VOS pool metrics
  * \param poh	[OUT]	Returned pool handle
  *
  * \return              Zero on success, negative value if error
  */
 int
-vos_pool_open(const char *path, uuid_t uuid, unsigned int flags,
+vos_pool_open(const char *path, uuid_t uuid, unsigned int flags, void *metrics,
```

Comment:
Since the metrics seem to be almost always NULL where this is changed, I wonder if it might be better to add a new function (e.g. `vos_pool_open_metrics`) that includes the metrics param. We can then keep this name/signature the same for callers that don't care about metrics. Could have one of the functions wrap the other to avoid duplication.

Reply:
sure